src/AutorestSwift/View Models/PropertyViewModel.swift

```diff
@@ -34,15 +34,17 @@ struct PropertyViewModel {
     let name: String
     let comment: ViewModelComment
     let type: String
+    // default value of the proeprty
     let defaultValue: ViewModelDefault
+    // default value of the property in the init(). Valid values are either nil (for optoinal property) or "" (i.e. not specfied for required property in init() method.)
     let initDefaultValue: String
     let isDate: Bool
     let optional: Bool
     let className: String
 
     /// Initialize from Value type (such as Property or Parameter)
     init(from schema: Value) {
-        self.name = schema.name
+        self.name = schema.serializedName ?? schema.name
```

Comment:
Would it make sense to have an assertion that the name is not empty string? That way we are alerted to the problem during code gen instead of when we are looking at the generated code?